src/pages/help.js

```diff
@@ -61,7 +61,9 @@ export default function Help(props) {
         <div className="mainContainer documentContainer postContainer helpContainer">
           <div>
             <header>
-              <h1>Need help?</h1>
+              <h1>
+                <p className="headings">Need help?</p>
```

Comment:
AFAIR, putting a `p` inside `h1` is not recommended based on the [spec](https://www.w3.org/TR/2014/REC-html5-20141028/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements). Any chance to remove this? :)

Reply:
Yes I can remove that, Is it okay to change `fontsize` of `h1` element.

Reply:
Yeah, that's alright :) thanks.